sample/common/src/commonMain/kotlin/com/mohamedrejeb/richeditor/sample/common/richeditor/RichEditorScreen.kt

```diff
@@ -1,13 +1,43 @@
 package com.mohamedrejeb.richeditor.sample.common.richeditor
 
 import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
 import cafe.adriel.voyager.core.screen.Screen
 
 object RichEditorScreen: Screen {
 
     @Composable
     override fun Content() {
-        RichEditorContent()
+        var basicRichTextValue by remember {
+                    mutableStateOf("""<h1>Text</h1>
+            <a href="https://www.w3schools.com">Visit W3Schools</a><br>
+            <p><b>RichTextEditor</b> is a <i>composable</i> that allows you to edit <u>rich text</u> content.</p>
+            <a href="https://github.com/DAKSHSEMWAL/Compose-Rich-Editor">MDParserKit Core</a><br>
+            <a href="https://music.youtube.com/search?q=summer+high">https://music.youtube.com/search?q=summer+high</a><br>
+            
+            <h2>An Unordered HTML List</h2>
+
+            <ul>
+              <li><a href="https://github.com/DAKSHSEMWAL/Compose-Rich-Editor">MDParserKit Core</a></li>
```

Comment:
I think you shouldn't add links to a fork of the library as an example. It might be confusing.